pkg/config/config.go

```diff
@@ -486,14 +498,33 @@ func AddAgentVersionToDomain(domain string, app string) (string, error) {
 	return u.String(), nil
 }
 
+func getMainInfraEndpoint(config Config) string {
+	return getMainEndpoint(config, defaultInfraURLPrefix, "dd_url")
+}
+
+// getMainEndpoint implements the logic to extract the DD URL from a config, based on `site` and ddURLKey
+func getMainEndpoint(config Config, prefix string, ddURLKey string) (resolvedDDURL string) {
```

Comment:
Will we ever use this function with another `ddUrlKey` than `dd_url` ? Same thing for `prefix` and `defaultInfraURLPrefix`.

I feel like we could remove `getMainInfraEndpoint` function and directly call `getMainEndpoint` with just the `config` parameter.

Reply:
(see my previous comment :) )